libqtile/layout/base.py

```diff
@@ -298,6 +298,20 @@ def remove(self, win):
                 focus = layouts[idx].focus_first()
         return focus
 
+    def show(self, screen):
+        raise NotImplementedError
```

Comment:
Should it has `@abstractmethod`? If so, we should add a docstring to explain to layout developers what they are meant to implement here, and maybe pass rather than breaking, like others abstract methods above.

Reply:
Yes. Thanks for catching that. How's this?

Reply:
Great, thanks!